ovos_plugin_common_play/ocp/gui.py

```diff
@@ -45,8 +47,36 @@ def bind(self, player):
                               self.handle_play_from_search)
         self.player.add_event('ovos.common_play.skill.play',
                               self.handle_play_skill_featured_media)
+        self.player.add_event('gui.page_gained_focus',
```

Comment:
for reference, this is emitted here https://github.com/OpenVoiceOS/ovos-core/blob/dev/mycroft/gui/bus.py#L130